crates/infra/utils/src/errors/mod.rs

```diff
@@ -113,13 +114,12 @@ impl ErrorDescriptor {
     fn write_problem_matcher(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         writeln!(
             f,
-            "slang-problem-matcher:{file}:{line}:{column}-{end_line}:{end_column}: {severity}: {message}",
+            "slang-problem-matcher:{file}:{line}:{column}-{end_line}:{end_column}: error: {message}",
```

Comment:
I suggest keeping the `{severity}` arg, since it makes it clear that this is an actual argument to the matcher pattern, and can only have specific values.

Reply:
Makes sense, I've extracted this in c160443b6ba9c451696f4cb665d5bc4ae77162ae